src/styles/utilities/_spacers.scss

```
@@ -51,7 +51,7 @@ $mt: (
 	"64": $spacer-lg * 2,
 	"80": $spacer-2xl,
 );
-@include generate-utils("mt", ("margin-top"), $mt);
+@include generate-utils("mt", ("margin-top"), $mt, ("md"));
```

Question: Are these extra parentheses needed? Or is this also valid?
```suggestion
@include generate-utils("mt", "margin-top", $mt, "md");
```